crates/net/network/src/peers/manager.rs

```diff
@@ -734,6 +736,8 @@ pub struct Peer {
     kind: PeerKind,
     /// Counts number of times the peer was backed off   
     backoff_counter: u32,
+    ///total time for the connection in active state in milliseconds
+    connection_time: Duration,
```

Comment:
we need to track an `Instant` here, which is the timestamp of the last tick

Reply:
okay I'll do the needful